content/docs/guide-development.md

```diff
@@ -24,13 +24,26 @@ You can use a mock [lightcurve notebook](https://renkulab.io/gitlab/astronomy/mm
 ### Parametetrize the notebook 
 
 * create a cell with the following tag "parameters" (see [papermill manual](https://papermill.readthedocs.io/en/latest/usage-parameterize.html#designate-parameters-for-a-cell)):
+  * if not annotated, the types of the inputs parameters are determined based on the parameter default value
+  * one can annotate the input parameter by putting comment with the `term` from the [ontology](https://odahub.io/docs/guide-ontology).
+  * several parameters are mandatory in the MMODA platform. These includes:
+    | Type annotation | Parameter default name |
+    | ---------------- | -------------- |
+    | http://odahub.io/ontology#PointOfInterestRA | RA |
```

Comment:
we should add them to the ontology https://github.com/oda-hub/hugo-odahub/issues/9